### DISCUSSION TITLE
How to create google meet link without going to oauth conscent screen.

### DISCUSSION THREAD

**Discussion by MOIN-AKHTAR** [googleapis/googleapis#813] [Q&A]
I have a scnerio inwhich i want to create an appointment between two user but the service i am using resolve all availability of two persons issue but it can't create meeting link for that i want to use google calanders api but for calander api i need to go through conscent screen then i'll get code and using this code i have to get first access token and refresh token after that i was able to create event and can create google meet link but i don't want user to go to cnsent screen how i can acheive this help plz.

**Comment by reallyterris** [comment]
You should call the person and here them write it down lol 


Sent from Yahoo Mail for iPhone


On Friday, June 16, 2023, 12:10 AM, Moin Akhter ***@***.***> wrote:




I have a scnerio inwhich i want to create an appointment between two user but now i the service i am using service resolve all availability of two persons issue but it can not create meeting link for that i want to use google calanders api but for calander api i need to go through conscent screen then i'll get code and using this code i have to get first access token and refresh token after that i was able to create event and can create google meet link but i don't want user to go to cnsent screen how i can acheive this help plz.

**Comment by enkhjargal** [comment]
It's not possible due to Google's security features.
